docs/developer_guide.md

```diff
@@ -239,32 +239,30 @@ the same region.
         }
       }
     ```
-  - If new stat vars are introduced, also update these files:
-    - static/js/i18n/strings/en/stat*var*[labels|titles].json.
-  - If adding new units, also add translations to ___
 
-- Update related files.
+1. Update related files.
+   - If adding a new category, create a new config file in [server/chart_config](../server/chart_config) and:
+     - [static/js/shared/util.ts](../static/js/shared/util.ts)
+     - [server/__init__.py](../server/__init__.py)
 
-  - If adding a new category, create a new config file in [server/chart_config](../server/chart_config) and:
-    - [static/js/shared/util.ts](../static/js/shared/util.ts)
-    - [server/__init__.py](../server/__init__.py)
+   - If a new stat var is introduced, also update:
+     [en/stats_var_labels.json](../static/js/i18n/strings/en/stats_var_labels.json) and [en/stats_var_titles.json](../static/js/i18n/strings/en/stats_var_titles.json)
 
-  - If a new stat var is introduced, also update:
-    [en/stats_var_labels.json](../static/js/i18n/strings/en/stats_var_labels.json) and [en/stats_var_titles.json](../static/js/i18n/strings/en/stats_var_titles.json)
+   - If a new unit is required, update:
+     - [i18n.tsx](../static/js/i18n/i18n.tsx)
+     - [en/units.json](static/js/i18n/strings/en/units.json) (with display names and labels for the unit)
 
-  - If a new unit is required, update:
-    - [i18n.tsx](../static/js/i18n/i18n.tsx) as well as
-    - [en/units.json](static/js/i18n/strings/en/units.json) with display names and labels for the unit.
+   Note: Please add very detailed descriptions to guide our translators. See localization.md for more details.
 
-  Note: Please add very detailed descriptions to guide our translators. See localization.md for more details.
+1. Run these commands:
```

Comment:
maybe emphasize this is required for every change above

Reply:
good point, updated